esp8266/qstrdefsport.h

```diff
@@ -42,11 +42,23 @@ Q(sync)
 Q(hard_reset)
 
 Q(esp)
+Q(esp.wifi)
```

Comment:
Does this work, having a dot in the name?  It's certainly not intended to work!  @pfalcon can we allow this and make it a "feature" of the way import works?